### ISSUE TITLE
arm install issue 

### ISSUE THREAD

**Issue by ghost** [rust-lang/rustup#1066]
When trying to install on ARM, this error occurs on the first line of install prompt

```sh: 249: [: unknown-linux-gnu: unexpected operator```

and on last 4 lines 

```
info: syncing channel updates for 'stable-armv7-unknown-linux-gnueabihf'
error: could not download file from 'https://static.rust-lang.org/dist/channel-rust-stable.toml.sha256' to '/home/pi/.rustup/tmp/1rc0k7s_gfvzw4r4_file'
info: caused by: error during download
info: caused by: [1] Unsupported protocol (Protocol "https" not supported or disabled in libcurl)
```

**Comment by malbarbo** [comment]
See https://github.com/rust-lang-nursery/rustup.rs/issues/1047 and https://github.com/rust-lang-nursery/rustup.rs/issues/1051

**Comment by zaynetro** [comment]
Using hyper fixed the issue (as in #1051):

```
export RUSTUP_USE_HYPER=1
curl https://sh.rustup.rs -sSf | sh
```

**Comment by Diggsey** [comment]
Closing as duplicate of mentioned issues.
